code/__DEFINES/research/research_categories.dm

```diff
@@ -75,6 +75,7 @@
 #define RND_CATEGORY_EQUIPMENT "/Equipment"
 #define RND_SUBCATEGORY_EQUIPMENT_SERVICE "/Service Equipment"
 #define RND_SUBCATEGORY_EQUIPMENT_JOKE "/Top-Secret Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_CARGO "/Cargo and Mining Equipment" // SkyRat Edit - why was this not here before?
```

Comment:
I feel like if you are going to add a whole category you should do this upstream.

Reply:
I did actually -i have a similar PR upstream to fix all of their categories

Reply:
i'll just wait for that to merge